app/views/posts/index.html.erb

```diff
@@ -4,6 +4,39 @@
     <title>Posts</title>
   </head>
   <body>
-    <p>here is a list of posts</p>
+    <section>
+      <% if @user.photo %>
+        <%= image_tag @user.photo, size: "40x40" %>
+      <% end %>
+      <p><%= @user.name %></p>
+      <p>Number of posts: <%= @user.posts_counter %></p>
+
+      <div>
+        <h3>Posts</h3>
+        <ul>
+          <% @user.posts.each do |post| %>
+            <li>
+              <a href="<%= user_post_path(@user, post) %>">
+                <h4><%= post.title %></h4>
+                <p><%= post.text %></p>
+                Comments:<%= post.comments_counter %>, Likes:<%= post.likes_counter %>
+                <div>
+                  <ul>
+                    <% post.five_most_recent_comments.each do |comment| %>
+                      <li>
+
+                        <p><%= comment.author.name %>: <%= comment.text %></p>
+                      </li>
+                    <% end %>
+                  </ul>
+                </div>
+              </a>
+
+            </li>
+          <% end %>
+        </ul>
+      </div>
+      <%= link_to 'See all posts', user_posts_path(@user) %>
+    </section>
```

Comment:
According to the [project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md)

Please let's modify the layout of all the `Views` to match the given *[wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md)*, as styling is nice to have 😊


Example

Your design
![image](https://github.com/mohamedSabry0/blog_hub/assets/78251927/cd9a5308-0d0d-47d2-87a9-8e504ec2d54d)

Wireframe
![image](https://github.com/mohamedSabry0/blog_hub/assets/78251927/26ac3c88-15e5-4ffe-94ea-63e4bfc4dbf3)